.doc_gen/templates/zonbook/service_chapter_template.xml

```diff
@@ -26,6 +29,27 @@
     </info>
     <para>The following code examples show how to use {{.ServiceEntity.Short}} with an &AWS; software development kit (SDK).
         {{.ServiceEntity.Caveat}}</para>
+    <variablelist role="termdef">
+        <para>The examples are divided into the following categories:</para>
+        {{- if $actions.Examples}}
+        <varlistentry>
+            <term>Actions</term>
+            <listitem><para>Excerpts that illustrate how to write code to call individual service functions.</para></listitem>
```

Comment:
<listitem><para>Code excerpts that show you how to call individual service functions.</para></listitem>

Reply:
Done